src/pkg/services/m365/api/graph/errors.go

```diff
@@ -241,14 +222,12 @@ func IsErrConnectionReset(err error) bool {
 func IsErrUnauthorizedOrBadToken(err error) bool {
 	return clues.HasLabel(err, LabelStatus(http.StatusUnauthorized)) ||
 		hasErrorCode(err, invalidAuthenticationToken) ||
-		errors.Is(err, ErrTokenExpired) ||
-		errors.Is(err, ErrTokenInvalid)
+		errors.Is(err, ErrTokenExpired)
 }
 
 func IsErrBadJWTToken(err error) bool {
 	return hasErrorCode(err, invalidAuthenticationToken) ||
-		errors.Is(err, ErrTokenExpired) ||
-		errors.Is(err, ErrTokenInvalid)
+		errors.Is(err, ErrTokenExpired)
```

Comment:
`ErrTokenExpired` can be removed from this. We created this error to indicate download URL jwt expiry. Adapter `Send` where `IsErrBadJWTToken` gets used doesn't do an inspection of download URL expiries.